doc(release) release notes web 23.10.20 - DSM 23.10.2 (#4026)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.10/releases/centreon-os.mdx

## Centreon Web

### 23.10.20

Release date: `January 27, 2025`

<details open>
  <summary>Enhancements</summary>

- [API] Added missing configuration change logs for host severities configuration.
- [API] Added missing configuration change logs for service severities configuration.
- [API] Added missing configuration change logs for host template configuration.
- [API] Added missing configuration change logs written when the configuration of services is deployed.

</details>

<details>
  <summary>Bug fixes</summary>

- [Authentication] Fixed an issue preventing LDAP users with more than one email address to log in to Centreon.

</details>

<details>
  <summary>Security fixes</summary>

> - [Security] Fixed a vulnerability in contact form.

Reviewer: ```suggestion
- [Security] Fixed a vulnerability in the contact creation form.
```

- [Security] Fixed SQLi in "Host Monitoring" widget in Custom Views, for authenticated users having sufficient rights to export data in CSV.
- [Security] Fixed SQLi in "Hostgroup Monitoring" widget in Custom Views, for authenticated users having sufficient rights to export data in CSV.
- [Security] Fixed SQLi in "Service Monitoring" widget in Custom Views, for authenticated users having sufficient rights to edit the widget.

> - [Security] Fixed a vulnerability in contact form.

Reviewer: ```suggestion
```

> - [Security] Updated Symfony Security-Http component.

Reviewer: ```suggestion
- [Security] Updated Symfony **Security-Http** component.
```

## Centreon DSM

### 23.10.2

Release date: `January 27, 2025`

<details>
  <summary>Bug fixes</summary>

> - [DSM] Fixed white page issue when accessing to "Administration > Extensions > Configure" menu.

Reviewer: ```suggestion
- [DSM] Fixed an issue where a blank page could be displayed when accessing the "Administration > Extensions > Configure" menu.
```
